Tests/Common/TestUtils.cs

```diff
@@ -297,5 +301,15 @@ public static void StartBasicsStation(TestConfiguration config, Dictionary<strin
                 throw new TimeoutException("Executing the SSH command took more than expected.");
             }
         }
+
+        public static RadioMetadata GenerateTestRadioMetadata(
+                DataRateIndex dataRate = DataRateIndex.DR2,
+                double frequency = 868.3,
+                uint antennaPreference = 1,
+                ulong xtime = 100000,
+                uint gpstime = 0,
+                double rssi = 2.0,
+                float snr = 0.1f
+            ) => new RadioMetadata(dataRate, Hertz.Mega(frequency), new RadioMetadataUpInfo(antennaPreference, xtime, gpstime, rssi, snr));
```

Comment:
Use `Hertz` instead of `double` for `frequency` parameter:

```suggestion
        public static RadioMetadata GenerateTestRadioMetadata(
                DataRateIndex dataRate = DataRateIndex.DR2,
                Hertz? frequency = null,
                uint antennaPreference = 1,
                ulong xtime = 100000,
                uint gpstime = 0,
                double rssi = 2.0,
                float snr = 0.1f) =>
            new RadioMetadata(dataRate, frequency ?? Hertz.Mega(868.3),
                              new RadioMetadataUpInfo(antennaPreference, xtime, gpstime, rssi, snr));
```

1\. This belongs with suggestion 2.